sdk/python/kfp/client/client.py

```diff
@@ -303,14 +303,10 @@ def _load_config(
                 namespace)
         return config
 
-    def _is_inverse_proxy_host(self, host):
+    def _is_inverse_proxy_host(self, host: str) -> bool:
         if host:
-            return re.match(r'\S+.googleusercontent.com/{0,1}$', host)
-        if re.match(r'\w+', host):
-            warnings.warn(
-                f'The received host is {host}, please include the full endpoint'
-                ' address (with ".(pipelines/notebooks).googleusercontent.com")'
-            )
+            if re.match(r'\S+.googleusercontent.com/{0,1}$', host) is not None:
```

Comment:
Used the negative to consolidate the `return False` statements into one, i.e. we return `False` if the regex doesn't match _or_ if the host is falsey.